snippets/auth/index.py

```diff
@@ -192,6 +192,26 @@ def get_user_by_email():
     print('Successfully fetched user data: {0}'.format(user.uid))
     # [END get_user_by_email]
 
+def bulk_get_users():
+    # [START bulk_get_users]
+    from firebase_admin import auth
+
+    result = auth.get_users([
+        auth.UidIdentifier('uid1'),
+        auth.EmailIdentifier('user2@example.com'),
+        auth.PhoneIdentifier(+15555550003),
+        auth.ProviderIdentifier('google.com', 'google_uid4')
+    ])
+
+    print('Successfully fetched user data:')
+    for user in result.users:
+        print(user)
```

Comment:
Same problem as in java... we haven't defined `__str__()` on our objects, so the results of these print statements aren't great. You could use the same solution, i.e.: `print(user.uid)`